Client/src/Components/DatasetUpload/DatasetValidationSchema.tsx

```diff
@@ -28,9 +28,9 @@ const referenceValidationSchema = Yup.object().shape({
 
 export const variableValidationSchema = Yup.object().shape(
   {
-    name: Yup.string().trim().required(requiredMessage('Name')),
-    repr: Yup.string().trim().required(requiredMessage('Representation')),
-    units: Yup.string().trim().required(requiredMessage('Units'))
+    name: Yup.string().trim(),
```

Comment:
You can remove this validation schema. Were not gonna use this formatting anymore

Reply:
@GD-Dheer you want me to remove the entire file?

Reply:
We're gonna get back to this in the units epic, I asked Leslie to make it like this

Reply:
So leave this as it is tight now

Reply:
Thanks for this change, while not directly related to the PR itself, this has been a slight headache in testing

Reply:
yeah i will merge without deleting this